.github/workflows/scripts/code_coverage_disable_check.py

```diff
@@ -37,11 +37,13 @@ def has_code_coverage_disable(file_path):
         bool: True if code coverage disable statement is found, False
         otherwise.
     """
-    code_coverage_disable_pattern = re.compile(
-        r"""//?\s*istanbul\s+ignore(?:\s+(?:next|-line))?[^\n]*|
-        /\*\s*istanbul\s+ignore\s+(?:next|-line)\s*\*/""",
-        re.IGNORECASE,
-    )
+   code_coverage_disable_pattern = re.compile(
+    r"""//?\s*istanbul\s+ignore(?:\s+(?:next|line)(?:\s*--\s*\S*)?)?[^\n]*|
+    /\*\s*istanbul\s+ignore(?:\s+(?:next|line)(?:\s*--\s*\S*)?)?\s*\*/""",
+    re.IGNORECASE,
```

Comment:
1. This should fail for any string that starts with `/* instanbul` and ends with `*/`
2. That will cover any future changes to the syntax